### ISSUE TITLE
表不存在但插入数据却成功返回交易id

### ISSUE THREAD

**Issue by ByCnck** [AschPlatform/asch#101]
![](http://asch-public.oss-cn-beijing.aliyuncs.com/pics/dapp%E6%A0%B8%E5%BF%83%E5%BC%80%E5%8F%91%E6%B5%81%E7%A8%8B%E8%A7%A3%E6%9E%90/15.png)

![](http://asch-public.oss-cn-beijing.aliyuncs.com/pics/dapp%E6%A0%B8%E5%BF%83%E5%BC%80%E5%8F%91%E6%B5%81%E7%A8%8B%E8%A7%A3%E6%9E%90/16.png)

但model下没有定义articles表，日志是有报错的
```
2018-01-25T15:33:40+0800 <debug> smartdb.js:1 (e.<anonymous>) execute sql: update "balances" set "balance" = '9970000000' where "address" = 'AAjoobuMcmkQ1gS8vTfBy3dQavBiH7sBCF' and "currency" = 'XAS';insert into "variables" ("key", "value") values ('article_max_id', '1');insert into "articles" ("title", "url", "text", "tags", "id", "votes", "tid", "authorId", "timestamp", "comments") values ('news_title', '', 'This is news text.', 'tag1', '1', 0, '830e8ff3e41f3f5b1b45689ac9917f487a7588f7b19ea708cf070f920419fe0a', 'AAjoobuMcmkQ1gS8vTfBy3dQavBiH7sBCF', 49808013, 0);insert into "round_fees" ("round", "currency", "amount") values (129, 'XAS', '10000000');insert into "transactions" ("fee", "senderPublicKey", "senderId", "timestamp", "type", "args", "signature", "id", "height") values ('10000000', '0926b2269d1f6b7f194ef57c548cc7a3f340fb06156ea31014ab7bd9e562fb15', 'AAjoobuMcmkQ1gS8vTfBy3dQavBiH7sBCF', 49808013, 1000, '["news_title","","This is news text.","tag1"]', '42855a25021c552a10a2a85ba27a01b9257ea0033957e5e9cb74cf7d2fb63d6779aef6195498219d1bbe5db28ab5ce9622a4bb6634481dbffe6ae11f37f79b09', '830e8ff3e41f3f5b1b45689ac9917f487a7588f7b19ea708cf070f920419fe0a', 643);insert into "blocks" ("delegate", "height", "prevBlockId", "pointId", "timestamp", "pointHeight", "count", "payloadHash", "payloadLength", "signature", "id") values ('39c2322600a0c81ecfa97119ec8e2d5bfb73394914d92b54e961846a987e4e22', 643, '0d75f297cda7deb6adb1301076199b7486efa9a73ea09d84798fdf18facbb8c7', '5e4cb60622dcefc5a2fc3d6dfecce4aa66ed8d0c435522d0c9ecaea4d1e16847', 49808020, 1818, 1, 'f5fa87dfbd598fc6bb3f70cd7847b5723a0cb770be68f10835de226ecc5bd66e', 93, 'df2fbcffcc42d039d7b6838ba5d972c63279ca32d81f9566d706c2ffdc15825f1f0309d9e257864454963bec05044356769676ac1240d4a7e55c092f26bd0d0f', 'b05070f41e52299965abc6bd8ba37dd5ae96a5701e0d142aef4c55c26cfa0357');
2018-01-25T15:33:40+0800 <error> smartdb.js:1 (e.<anonymous>) -!!!!!!!!!!!!!!!!!!!!!!commit error Error: Error: near line 628: no such table: articles

    at y (/mnt/d/Codes/github/AschPlatform/asch/node_modules/asch-sandbox/framework/helpers/dblite.js:1:2434)
    at Socket.<anonymous> (/mnt/d/Codes/github/AschPlatform/asch/node_modules/asch-sandbox/framework/helpers/dblite.js:1:2588)
    at emitOne (events.js:115:13)
    at Socket.emit (events.js:210:7)
    at addChunk (_stream_readable.js:266:12)
    at readableAddChunk (_stream_readable.js:253:11)
    at Socket.Readable.push (_stream_readable.js:211:10)
    at Pipe.onread (net.js:585:20)
2018-01-25T15:33:40+0800 <error> blocks.js:1 (Blocks.<anonymous>) save block error:  Error: Failed to commit block: Error: Error: near line 628: no such table: articles
```

**Comment by a1300** [comment]
May I ask which dapp-template you are using?

Are you using [sqfasd/asch-hello-dapp](https://github.com/sqfasd/asch-hello-dapp) or maybe  [aschplatform/asch-hello-dapp](https://github.com/sqfasd/asch-hello-dapp) ? :-)

Thank you!

**Comment by yanyiwu** [comment]
@a1300 i think https://github.com/sqfasd/asch-hello-dapp is deprecated.

**Comment by yanyiwu** [comment]
Dear @a1300,
i have a suggestion, if you have any question,  posting a new issue is better than commenting on the unrelated issue. 
Thank you.

**Comment by a1300** [comment]
Dear @yanyiwu,

in the future I will create a new issue :-)

Kind regards
a1300

**Comment by eagleHovering** [comment]
It's fixed in new v1.3.6
